requirements.txt

```diff
@@ -28,16 +29,19 @@ flower==0.9.1
 jsonschema==2.6.0
 Markdown==2.2.1
 mock==2.0.0
-mockcache==1.0.2
+mod_wsgi==4.6.7
+git+git://github.com/lunant/mockcache@master
```

Comment:
The last commit to that repo was in 2016. I looks like this module is no longer maintained, so need to find a replacement or remove it.

Reply:
Nils and I both felt comfortable pinning it to master (we can also do a specific commit) but removing it should be a separate issue independent of this pull request if you feel strongly.

Reply:
Does Nils know that this package is no longer maintained?

Reply:
yes - we only use this in our unit testing